templates/azure/support.html

```diff
@@ -1,145 +1,164 @@
-      {{ image (
-        url="https://assets.ubuntu.com/v1/d649019e-u+pro+logo+158x40.png",
-        alt="",
-        width="395",
-        height="100",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
-    </div>
-  </div>
-  <div class="u-fixed-width">
-    <hr class="p-separator" />
-  </div>
-  <div class="row">
-    <div class="col-6">
-      <h2>Ubuntu Pro with 24x7 Support</h2>
-      <p>With 24x7 SLAed support and the widest open source security patching, teams can build on Ubuntu Pro with Support to secure their application's open source supply chain and receive expert assistance when they need it.</p>
-      <p>24x7 Support is available for the Ubuntu operating system (infrastructure) and can be extended to include application support for <a href="/support#apps-support">key open source applications</a>.</p>
-      <p>See more details at <a href="/support">https://ubuntu.com/support</a>.</p>
-      <p>If you already purchase support from Azure, speak to us about adding and integrating Canonical's Support into your existing Azure support &mdash; ensuring you have vendor-backed, SLAed support above the hypervisor for your workloads, without having to make any changes to end user issue reporting.</p>
-    </div>
-    <div class="col-6 u-align--center u-vertically-center u-hide--medium u-hide--small">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/e9d3cf66-We+support+your+operations.svg",
-        alt="",
-        width="150",
-        height="150",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
+    <section class="p-section">
+      <div class="row--50-50">
+        <hr />
+        <div class="col">
+          <h2>Ubuntu Pro</h2>
+        </div>
+        <div class="col">
+          <p>
+            Ubuntu Pro for Azure is a premium image delivering the most comprehensive open source security and Azure compliance. Ubuntu Pro is suitable for small to large-scale Linux enterprise operations offering pay-as-you-go billing on your existing Azure invoice.
+          </p>
+          <p>
+            For more information about Ubuntu Pro on Azure, see <a href="/azure/pro">https://ubuntu.com/azure/pro</a>.
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section class="p-section">
+      <div class="row--50-50">
+        <hr />
+        <div class="col">
+          <h2>Ubuntu Pro with 24x7 Support</h2>
+        </div>
+        <div class="col">
+          <p>
+            With 24x7 SLAed support and the widest open source security patching, teams can build on Ubuntu Pro with Support to secure their application's open source supply chain and receive expert assistance when they need it.
+          </p>
+          <p>
+            24x7 Support is available for the Ubuntu operating system (infrastructure) and can be extended to include application support for <a href="/support#apps-support">key open source applications</a>.
```

Comment:
```suggestion
            24x7 Support is available for the Ubuntu operating system (infrastructure) and can be extended to include application support for <a href="/support">key open source applications</a>.
```
This anchor doesn't lead anywhere. Either remove it or make it point to something